include/amici/model.h

```diff
@@ -260,16 +262,20 @@ class Model : public AbstractModel, public ModelDimensions {
 
     /**
      * @brief Initialize initial states.
+     * @param t Timepoint
```

Comment:
Rename to t0? Same below

Reply:
`Model::{x0,sx0}` use `t`. No strong opinion.